### PR TITLE
Import model

### PR DESCRIPTION
**Pull request by phildow**
There might still be some refactoring to do but I'd love to get this on in-office devices so that we can start testing locally.

### CODE REVIEW

File: Net Runner/Settings/AddModelTableViewController.mm [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+    
+    dispatch_async(dispatch_get_main_queue(), ^{
+        self.downloadProgressView.hidden = NO;
+        self.downloadProgressView.progress = 0;
+        self.downloadLabel.hidden = NO;
+    });
+}
+
+- (void)modelImporter:(ModelImporter*)importer downloadDidProgress:(float)progress {
+    NSLog(@"Importer Did Progress");
```
Comment (nkashy1) [inline Net Runner/Settings/AddModelTableViewController.mm]: Getting failures with the following message:
```
2018-09-25 15:21:35.426042-0700 Net Runner[58280:1198866] Task <0A2FBCEA-D8BD-4811-9A24-1D6001AB31E9>.<1> finished with error - code: -1002
2018-09-25 15:23:51.219895-0700 Net Runner[58280:1191531] Importer Did Begin Download
2018-09-25 15:23:51.220704-0700 Net Runner[58280:1199591] Task <5F87FC8D-70F2-43FF-BB19-F6BEBF50463C>.<1> finished with error - code: -1002
2018-09-25 15:26:35.202885-0700 Net Runner[58280:1191531] Importer Did Begin Download
2018-09-25 15:26:35.203130-0700 Net Runner[58280:1203627] Task <1DAE37BE-A89F-4439-8F62-56ECB5E95F66>.<1> finished with error - code: -1002
```

Comment (nkashy1) [reply Net Runner/Settings/AddModelTableViewController.mm]: The issue with those seems to have been that I was exposing the zipfile using Python's `SimpleHTTPServer` over `localhost` and the simulated iPhone does not understand how to look up on `localhost`?

Putting it on tinyupload led to an error saying that the file could not be unzipped.

I created the zipfile as follows:
1. I took the phenomenal-face model, renamed it to Dark face, and kept everything else as is.
2. I renamed the tfbundle directory to dark-face.tfbundle
3. I ran `zip -r dark-face.tfbundle.zip dark-face.tfbundle/*`

This seems not to work. I will try again using the Finder compress option. Please add a command line equivalent, though.

Comment (nkashy1) [reply Net Runner/Settings/AddModelTableViewController.mm]: Doing this with a finder-compressed file led to the same error that I was seeing earlier:
```
2018-09-25 15:38:07.806290-0700 Net Runner[58280:1191531] Importer Did Begin Download
2018-09-25 15:38:07.806718-0700 Net Runner[58280:1211738] Task <EFD8FF0B-9660-43CA-BE30-8F360386D5E3>.<1> finished with error - code: -1002
```

This suggests that I input the URL incorrectly. The error should surface in the UI.

Comment (nkashy1) [reply Net Runner/Settings/AddModelTableViewController.mm]: The upload is not working with the model uploaded here: https://files.fm/u/9d936z3k

@phildow : You can use that to debut, I may have made some rookie mistake.

We should not merge until this experience is better.

Comment (phildow) [reply Net Runner/Settings/AddModelTableViewController.mm]: @nkashy1 I was always testing with SimpleHTTPServer on localhost and it worked fine on my end in the simulator. I zipped the .tfbundle folder itself. And errors should definitely surface to the UI, not sure why this one isn't. Let me try with that file on my end.

**Comment by phildow** [comment]
@nkashy1 regarding the other comments, all those sound good:

- Update README
- Add ability to delete a model
- Show model.json in model details

Regarding the download failure, that error code -1002 is surfacing from the http api. Are you cancelling the download when it gets stuck? Or are there any other network issues. I was actually able to downloading and import the file you mention above at the following modified url:

https://files.fm/down.php?cf&i=mb37grec&n=dark-face.tfbundle.zip

File: Net Runner/Settings/AddModelTableViewController.mm (hunk 1 above)
Comment (nkashy1) [reply Net Runner/Settings/AddModelTableViewController.mm]: Alright, there were a lot of mistakes on my side:
1. I hadn't `pod install`ed the latest TensorIO or the zip library
2. I missed the `http://` prefix on the model URL
3. The files.fm link downloaded an HTML page that provided a download button for the file (so was not a good test).

At the end of the day, running `SimpleHTTPServer` worked, for a zipfile made by the Finder compress as well as a zipfile made with the `zip` command line utility.

The only thing to change, I would say, is validate that there is an `http://` or `https://` prefix in the tfbundle zip URL.

**Comment by phildow** [comment]
@nkashy1 take another look when you get the changes and please accept the changes. Should be good to go!

**Pull request merged**